modules/ROOT/pages/CommunityResources/incredibuild.adoc

```diff
@@ -0,0 +1,111 @@
+= Setting Up Incredibuild
+
+Satisfactory is a large game.
+As such, it can take an enormous amount of time to compile the relevant, even on high end hardware.
+In this section, we will guide you through the process of setting up Incredibuild,
+which is a build acceleration tool for Microsoft Visual Studio and Jetbrains Rider.
+It can significantly reduce the amount of time it takes to compile the modding project,
+which is especially helpful if you're doing a lot of C++ development.
+
+== How it works
+
+Incredibuild works by using some clever compiler optimizations,
```

Comment:
```suggestion
```
I don't think it does. Running distributed builds (either with IB, FASTBuild, SNDBS, Bazel, whatever) on a single machine is generally a bit slower, there's overhead